### ISSUE TITLE
ModuleNotFoundError: No module named 'importlib.resources'

### ISSUE THREAD

**Issue by simahawk** [akretion/factur-x#48]
```
2024-12-03 15:29:44,540 2210 ERROR odoo odoo.tests.loader: Can not `import account_invoice_facturx`. 
Traceback (most recent call last):
  File "/opt/odoo/odoo/tests/loader.py", line 29, in _get_tests_modules
    mod = importlib.import_module('.tests', modpath)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/__w/edi/edi/account_invoice_facturx/tests/__init__.py", line 1, in <module>
    from . import test_facturx_invoice
  File "/__w/edi/edi/account_invoice_facturx/tests/test_facturx_invoice.py", line 7, in <module>
    from facturx import get_facturx_level
  File "/opt/odoo-venv/lib/python3.6/site-packages/facturx/__init__.py", line 1, in <module>
    from .facturx import generate_from_file, \
  File "/opt/odoo-venv/lib/python3.6/site-packages/facturx/facturx.py", line 37, in <module>
    import importlib.resources
ModuleNotFoundError: No module named 'importlib.resources'
```
Happens at least on Odoo 14.0 and 15.0.
https://github.com/OCA/edi/issues/1084

**Comment by simahawk** [comment]
Ok, looks like we have to pin a version prior to this change https://github.com/akretion/factur-x/pull/39

**Comment by alexis-via** [comment]
importlib.resources was added in python 3.7, cf
https://docs.python.org/3/library/importlib.resources.html#module-importlib.resources
The factur-x lib declares python >= 3.6 ; I updated it to 3.7.
I made a new release 3.4 with the fix.

OCA/edi need to pin an older version of the factur-x lib, or use a not-so-old version of python for it's tests !
The last ubuntu LTS version still under maintainence is 20.04 LTS and it is shipped with python 3.8. 3.6 is really old !
